tests/documentation/command-reference/doc_command_reference_init_test.go

```diff
@@ -0,0 +1,220 @@
+package docautomation
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/go-cmp/cmp"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhat-developer/odo/tests/helper"
+)
+
+var _ = Describe("doc command reference odo init", Label(helper.LabelNoCluster), func() {
+	var commonVar helper.CommonVar
+	var commonPath = filepath.Join("command-reference", "docs-mdx", "init")
+	var outputStringFormat = "```console\n$ odo %s\n%s```\n"
+
+	BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+		helper.Chdir(commonVar.Context)
+		Expect(helper.VerifyFileExists(".odo/env/env.yaml")).To(BeFalse())
+	})
+
+	AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+	Context("Interactive Mode", func() {
+		It("Empty directory", func() {
+			args := []string{"odo", "init"}
+			out, err := helper.RunInteractive(args, []string{"ODO_LOG_LEVEL=0"}, func(ctx helper.InteractiveContext) {
+				helper.ExpectString(ctx, "Select language")
+				helper.SendLine(ctx, "Java")
+
+				helper.ExpectString(ctx, "Select project type")
+				helper.SendLine(ctx, "")
+
+				helper.ExpectString(ctx, "Select container for which you want to change configuration?")
+				helper.SendLine(ctx, "")
+
+				helper.ExpectString(ctx, "Which starter project do you want to use")
+				helper.SendLine(ctx, "")
+
+				helper.ExpectString(ctx, "Enter component name")
+				helper.SendLine(ctx, "my-java-maven-app")
+
+				helper.ExpectString(ctx, "Your new component 'my-java-maven-app' is ready in the current directory")
+			})
+			Expect(err).To(BeNil())
+			got := fmt.Sprintf(outputStringFormat, " init", helper.StripSpinner(out))
+			got = helper.ReplaceTimeInString(got, "[1s]")
+			want := helper.GetMDXContent(filepath.Join(commonPath, "interactive_mode_empty_directory_output.mdx"))
+			want = helper.ReplaceTimeInString(want, "[1s]")
```

Comment:
You could get rid of this line, by setting [1s] in the mdx file once and for all